R/dplyr.R

```diff
@@ -47,12 +46,18 @@ filter.dm <- function(.data, ...) {
 
 #' @export
 filter.zoomed_dm <- function(.data, ...) {
-  quos <- enquos(...)
-  if (is_empty(quos)) {
+  filter_exprs <- enexprs(...)
+  if (is_empty(filter_exprs)) {
     return(.data)
   } # valid table and empty ellipsis provided
 
-  set_filter_for_table(.data, orig_name_zoomed(.data), quos, TRUE)
+  tbl <- get_zoomed_tbl(.data)
+  filtered_tbl <- filter(tbl, ...)
```

Comment:
In general, once we `enquo()` or `enexpr()`, we should use this and not pass on the ellipsis.

```suggestion
  filtered_tbl <- filter(tbl, !!!filter_quos)
```